src/mono/mono/mini/mini-arm64.c

```diff
@@ -4083,17 +4087,19 @@ mono_arch_output_basic_block (MonoCompile *cfg, MonoBasicBlock *bb)
 
 		case OP_ARM64_XADDV: {
 			switch (ins->inst_c0) {
-			case INTRINS_AARCH64_ADV_SIMD_FADDV:
+			case INTRINS_AARCH64_ADV_SIMD_FADDV: {
+				const int width = get_vector_size_macro (ins);
 				if (ins->inst_c1 == MONO_TYPE_R8) {
-					arm_neon_faddp (code, VREG_FULL, TYPE_F64, dreg, sreg1, sreg1);
+					arm_neon_faddp (code, width, TYPE_F64, dreg, sreg1, sreg1);
```

Comment:
The `faddp` instruction is not defined if `sz==1` and `Q==0`, i.e. when pairwise-adding doubles in a 64bit vector.

Reply:
You are right, I overlooked this one.